### PR TITLE
Viously Bid Adapter : New Adapter

### PR DESCRIPTION
**Pull request by geoffray-viously**
## Type of change
- [x] New bidder adapter

## Description of change

New Bid Adapter for Viously

Email : prebid@viously.com

Test parameters :
```
{
  bidder: 'viously',
  params: {
    pid: '20d30b78-43ec-11ed-b878-0242ac120002'
  }
}
```

## Other information

PR in prebid.github.io : https://github.com/prebid/prebid.github.io/pull/4058

### CODE REVIEW

**Review by osazos — COMMENTED** [review]
Hi @geoffray-viously 

During my tests, I got an error response from your endpoint. Can you check/fix it?

```
Request URL: https://bidder.viously.com/bid
Request Method: POST
Status Code: 405 
Referrer Policy: strict-origin-when-cross-origin

# payload
{
   "pid":"20d30b78-43ec-11ed-b878-0242ac120002",
   "domain":"adagio-local.io",
   "page_domain":"https://adagio-local.io:9999/integrationExamples/gpt/_outstream.html?pbjs_debug=true",
   "currency_code":"EUR",
   "placements":[
      {
         "id":"div-gpt-ad-1542984663788-0",
         "bid_id":"2741e71b4e6552",
         "video_params":{
            "context":"instream",
            "playbackmethod":[1, 2, 3, 4, 5, 6],
            "size":["400x300"]
         }
      }
   ]
}
```

**Comment by osazos** [comment]
ping @geoffray-viously

File: modules/viouslyBidAdapter.js [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+        playbackmethod: deepAccess(bidRequest, 'mediaTypes.video.playbackmethod'),
+        size: parseSizesInput(deepAccess(bidRequest, 'mediaTypes.video.playerSize'))
+      };
+
+      return request;
+    });
+
+    return {
+      method: HTTP_METHOD,
+      url: validBidRequests[0].params.endpoint ? validBidRequests[0].params.endpoint : REQUEST_URL,
```
[the remark this answers is not in the document]
Comment (geoffray-viously) [reply modules/viouslyBidAdapter.js:146]: Yes it is for internal debug usage for testing purposes.

**Comment by geoffray-viously** [comment]
Hi @osazos ,

Thank you for the review.
Concerning the endpoint, we're currently working on it and it should be available by the end of the week.

**Comment by osazos** [comment]
Ok @geoffray-viously, let me know when it is ok.

**Comment by geoffray-viously** [comment]
Hello @osazos , the endpoint is ready.
It will return an empty response though, as it only responds with an ad for real users.

**Review by osazos — APPROVED** [review]
LGTM

**Pull request merged**